Clover/app/src/main/java/org/floens/chan/ui/controller/ImageViewerController.java

```diff
@@ -534,4 +535,13 @@ private Window getWindow() {
     public interface GoPostCallback {
         ImageViewerCallback goToPost(PostImage postImage);
     }
+
+    /**
+     * Send thumbnail image of webm posts because none of the image search providers support webm directly
+     * @param postImage
+     * @return url of an image to be searched
+     */
+    private String getSearchImageUrl(final PostImage postImage) {
+        return postImage.extension.matches("^webm$") ? postImage.thumbnailUrl : postImage.imageUrl;
```

Comment:
I think `.matches("^webm$")` is the same as `.equals("webm")`.

Reply:
I'll do one better, I'll change it to postImage.type == PostImage.Type.MOVIE

Reply:
done